mappings/net/minecraft/util/OrderedSet.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/OrderedSet
+	CLASS class_4707 Itr
+		FIELD field_21566 start I
+		FIELD field_21567 index I
+	FIELD field_21562 sortingFunction Ljava/util/Comparator;
```

Comment:
This is like an internal logic for the array-backed set so that lookup performance can be O(N log(N)). Given there is a `first` method, I suggest calling this class a `PrioritySet` and this field a `comparator`.
```suggestion
	FIELD field_21562 comparator Ljava/util/Comparator;
```